apps/ledger-live-desktop/tests/specs/speculos/accountSpeculos.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { specs } from "../../utils/speculos";
+import { Currency } from "../../enum/Currency";
+
+const currencies: Currency[] = [
+  Currency.BTC,
+  Currency.tBTC,
+  Currency.ETH,
+  Currency.tETH,
+  Currency.XRP,
+  Currency.SOL,
+  Currency.DOT,
+  Currency.TRX,
+];
+
+for (const currency of currencies) {
+  test.describe.parallel("Accounts @smoke", () => {
+    test.use({
+      userdata: "skip-onboarding",
+      testName: `receiveSpeculos_${currency.uiName}`,
+      speculosCurrency: specs[currency.deviceLabel.replace(/ /g, "_")],
+      speculosOffset: Math.floor(Math.random() * 10000),
+    });
+    let firstAccountName = "NO ACCOUNT NAME YET";
+    let accountsListBeforeRemove: (string | null)[];
+    test(`[${currency.uiName}] Add account`, async ({ page }) => {
+      const portfolioPage = new PortfolioPage(page);
+      const addAccountModal = new AddAccountModal(page);
+      const layout = new Layout(page);
+      const accountsPage = new AccountsPage(page);
+      const accountPage = new AccountPage(page);
+
+      await test.step(`[${currency.uiName}] Open modal`, async () => {
+        await portfolioPage.openAddAccountModal();
+        expect(await addAccountModal.title.textContent()).toBe("Add accounts");
+        await expect(addAccountModal.selectAccount).toBeVisible();
+      });
+
+      await test.step(`[${currency.uiName}] Select currency`, async () => {
+        await addAccountModal.select(currency.uiName);
+        await page.mouse.move(0, 0); // prevent instability if select is hovered
+        await addAccountModal.continue();
+      });
+
+      await test.step(`[${currency.uiName}] Open device app`, async () => {
+        await addAccountModal.waitForSync();
+        const name = await addAccountModal.getFirstAccountName();
+        firstAccountName = name;
+        await expect(addAccountModal.addNewAccount).toBeVisible();
+      });
+
+      await test.step(`[${currency.uiName}] Scan and add accounts`, async () => {
+        await addAccountModal.addAccounts();
+        await expect(addAccountModal.successAdd).toBeVisible();
+      });
+
+      await test.step(`[${currency.uiName}] Done`, async () => {
+        await addAccountModal.done();
+        await layout.totalBalance.waitFor({ state: "visible" });
+      });
+
+      await test.step(`Navigate to first account`, async () => {
+        await layout.goToAccounts();
+        accountsListBeforeRemove = await accountsPage.getAccountsName();
+        await accountsPage.navigateToAccountByName(firstAccountName);
+        await expect(accountPage.accountName(firstAccountName)).toBeVisible();
+        await accountPage.settingsButton.waitFor({ state: "visible" });
+      });
+
+      await test.step(`scroll to operations`, async () => {
+        await accountPage.scrollToOperations();
+        await expect(accountPage.lastOperation).toBeVisible();
+      });
+
+      await test.step(`Delete current account`, async () => {
```

Comment:
why do you delete account at the end? I think it doesn't represent actual user's workflow

Reply:
For this test, I was inspired by what was already done in the Account test (specs/accounts/account)

Reply:
I think we should try to stick to what is described in test case in corresponding XRAY ticket, does it say create account and delete it?

Reply:
Ok, I'm going to create a new test case Delete Account 😀